libvirt/playbooks/download_zuul_ci_roles.yaml

```diff
@@ -0,0 +1,24 @@
+- name: clone git repos
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  vars:
+    git_repos:
+      - src: https://github.com/ansible/ansible-zuul-jobs.git
+        name: "ansible-zuul-jobs"
+      - src: https://opendev.org/zuul/zuul-jobs.git
+        name: "zuul-jobs"
+    ansible_zuul_dir: "../ansible-zuul"
```

Comment:
Since they repositories are handled by download_zuul_ci_roles, I would be prefer to store them in the current directory.
This way we avoid any surprise.

Reply:
yes of course we will then need to update the ansible.cfg file

Reply:
Yes indeed.